nntrainer/tensor/uint4_tensor.cpp

```diff
@@ -445,7 +445,7 @@ float Uint4QTensor::max_abs() const {
   uint8_t abs_max_val = 0;
   uint8_t curr_val;
   for (unsigned int idx = 0; idx < size(); ++idx) {
-    curr_val = std::abs(getValue(idx));
+    curr_val = getValue(idx);
```

Comment:
this function is max_abs() 
Using std::abs for comparison purposes seems correct, but are there any potential issues?